core/src/test/scala/org/broadinstitute/dsde/rawls/entities/compact/BatchUpsertScalingSpec.scala

```diff
@@ -1,4 +1,4 @@
-package org.broadinstitute.dsde.rawls.entities.local
+package org.broadinstitute.dsde.rawls.entities.compact
```

Comment:
I moved a few unit tests which are still valid for Quicksilver out of the "local" package and into the "compact" package